test/Microsoft.AspNetCore.Diagnostics.HealthChecks.Tests/HealthCheckMiddlewareTests.cs

```diff
@@ -395,6 +395,131 @@ public async Task CanListenWithoutPath_AcceptsRequest()
             Assert.Equal("Healthy", await response.Content.ReadAsStringAsync());
         }
 
+        [Fact]
+        public async Task CanListenWithPath_AcceptsRequestWithExtraSlash()
```

Comment:
Adding some additional tests here because we're not using `.Map` anymore, we're doing a semi-custom thing.